module/VuFind/src/VuFind/View/Helper/Bootstrap3/AvailabilityStatus.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * Helper class for rendering availability statuses.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Hebis Verbundzentrale 2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Thomas Wagener <wagener@hebis.uni-frankfurt.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace VuFind\View\Helper\Bootstrap3;
```

Comment:
Yes, I think we should move at least `renderAjax` to Root since it is not bootstrap specific. The classes in `getClass` are bootstrap classes but they are the same in bootstrap 3 and 5. So I'm not sure where to put this.

Reply:
One easy approach could be to move the whole thing into root, but to define the classes as properties. You could default them to bootstrap values since right now, those are the only values we need. But then it would be easy to create theme-specific overrides as needed.

Reply:
Good idea. I implemented this.